int64 flag
string[] obj_name
float64[] obj_pos
---
bool result
